code/modules/mob/living/carbon/human/human.dm

```diff
@@ -130,6 +130,7 @@
 		qdel(E)
 	for(var/obj/item/organ/O in internal_organs)
 		qdel(O)
+	death()

```

Comment:
Won't this cause a `seizes up lifeless` message when a human is deleted, like at round start for `robotize`